--- nodm-0.11.orig/Makefile.am
+++ nodm-0.11/Makefile.am
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-CFLAGS += -Wall -Werror
+CFLAGS += -Wall
 
 ACLOCAL_AMFLAGS = -I m4
 
@@ -12,7 +12,7 @@
 testlibsources = $(libsources) test.c
 
 AM_CPPFLAGS = $(X11_CFLAGS)
-AM_LDFLAGS = $(PAM_LIBS) $(X11_LIBS)
+LIBS = $(PAM_LIBS) $(X11_LIBS)
 
 nodm_SOURCES = $(libsources) nodm.c
 nodm_CFLAGS = -DNODM_SESSION='"$(sbindir)/nodm"'
